src/main/java/aquarium/shop/product/ProductController.java

```diff
@@ -0,0 +1,58 @@
+package aquarium.shop.product;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ProductController {
+
+    @Autowired
+    ProductService productService;
+
+
+    @GetMapping("/products")
+    public List<Product> getAllProducts() {
+        return productService.findAll();
+    }
+
+    @GetMapping({"products/{id}"})
+    public ResponseEntity<Product> getProduct(@PathVariable(value = "id") long id) {
```

Comment:
If in getAll method we simply return List<Product> I think here just **Product** should be returned, without using of ResponseEntity object.